includes/frontend.php

```diff
@@ -298,6 +298,14 @@ public function body_class( $classes = [] ) {
 			$classes[] = 'elementor-page elementor-page-' . $id;
 		}
 
+		$editor_preferences = SettingsManager::get_settings_managers( 'editorPreferences' );
+		
+		$show_hidden_elements = $editor_preferences->get_model()->get_settings( 'show_hidden_elements' );
+
+		if( 'yes' === $show_hidden_elements ){
+			$classes[] = 'show-hidden-elements';
+		}
+
```

Comment:
This should only happen in the Editor, not in the frontend.
Try adding a check for `Plugin::$instance->editor->is_edit_mode()`